banner-with-cta/client.js

```diff
@@ -13,23 +22,27 @@
          * Visit the Template Display Utilities documentation to learn more:
          * https://developer.evergage.com/templates/display-utilities
          */
-        return Evergage.DisplayUtils.pageElementLoaded(contentZoneSelector).then(element => {
+        return Evergage.DisplayUtils.bind(buildBindId(context)).pageElementLoaded(contentZoneSelector).then(element => {
             const html = template(context);
             Evergage.cashDom(element).html(html);
         });
     }
 
     function reset(context, template) {
-        Evergage.cashDom(`[data-evg-campaign-id=${context.campaign}][data-evg-experience-id=${context.experience}]`)
+        Evergage.DisplayUtils.unbind(buildBindId(context));
+        Evergage.cashDom(`[data-evg-campaign-id="${context.campaign}"][data-evg-experience-id="${context.experience}"]`)
```

Comment:
```suggestion
        Evergage.cashDom(`[data-evg-campaign-id=${context.campaign}][data-evg-experience-id=${context.experience}]`)
```
Revert back to original. Keep it consistent with syntax used in other templates